app/controllers/admin/categories_controller.rb

```diff
@@ -12,8 +12,13 @@ def new
 
   def create
     @category = ProductCategory.new(category_params)
-    @category.save!
-    redirect_to admin_categories_path
+    if @category.save!
```

Comment:
please check, if record does not save then it will go in else block? as you are saving record with "!".